doc/read-the-docs-site/reference/cardano/language-changes.rst

```diff
@@ -53,6 +53,48 @@ Vasil
 
 All of the built-in types and functions from ``PlutusV1`` were added to ``PlutusV2``.
 
-The following built-in function was added to ``PlutusV2`` only (i.e., it is not available in ``PlutusV1``).
+The following built-in function was added to ``PlutusV2`` only (ie, it is not available in ``PlutusV1``).
 
 - ``serializeData`` (proposed in `CIP-42 <https://cips.cardano.org/cips/cip42/>`_)
+
+PlutusV3
+~~~~~~~~~
+
+Plutus and cryptography teams at IOG, in collaboration with `MLabs <https://mlabs.city/>`_, continue to develop Plutus capabilities. Starting with the release of `Cardano node v.8.8.0-pre <https://github.com/IntersectMBO/cardano-node/releases/tag/8.8.0-pre>`_, ``PlutusV3`` is available on `SanchoNet <https://sancho.network/>`_, introducing the Cardano community to governance features from `CIP-1694 <https://cips.cardano.org/cip/CIP-1694#goal>`_ in a controlled testnet environment. 
```

Comment:
I suggest removing this paragraph. It's not very useful to talk about pre-releases and sanchonet, especially since `8.8.0-pre` doesn't yet support all the new primitives.